scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -172,12 +172,61 @@
       (cc/write-compute-cluster conn {:compute-cluster/type :compute-cluster.type/kubernetes
                                       :compute-cluster/cluster-name compute-cluster-name}))))
 
+(defn get-bearer-token
```

Comment:
API docs (here and the other functions). Perhaps a quick link to text why this is necessary, and/or a bit of expository text.

Reply:
Done